core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.Comparator;
+import java.util.List;
+
+class MultipleReturnValuesUsingApacheCommonsPair {
+
+    static ImmutablePair<Coordinates, Double> getMostDistantPoint(
+        List<Coordinates> coordinatesList,
+        Coordinates target) {
+        return coordinatesList.stream()
+                              .map(coordinates -> ImmutablePair.of(coordinates, coordinates.calculateDistance(target)))
```

Comment:
When continuing a line, we use a two space indent

Reply:
fixed all continuation indents, such as long method signatures.
It seems that my IDE (intellij) dosn't think the fluent function calls starting with dot as continuations. Should I make them have indent 2 anyway? (Personally I think the current format is easier to read)

Reply:
Yes ... two space indentation for line continuations. If we don't fix it now, we will have to fix it later :)

Reply:
Ok. thanks.
I fixed the indentation.